### PR TITLE
Country and Overview bug fixes

### PR DESCRIPTION
**Pull request by shreeyash07**
- Addresses  #412 #417 #418 #422 #419 

## Changes

* Format passed for outbreak overview chart
* Indicator wise heading in charts both in country and modal

## This PR doesn't introduce any:

- [x] temporary files, auto-generated files or secret keys
- [x] build works
- [x] eslint issues
- [x] typescript issues
- [x] codegen errors
- [x] `console.log` meant for debugging
- [x] typos
- [x] unwanted comments
- [x] conflict markers

### CODE REVIEW

File: app/views/Dashboard/Country/index.tsx [hunk 1]
```diff
@@ -751,7 +753,9 @@ function Country(props: Props) {
                     (group, key) => group.reduce(
                         (acc, item) => ({
                             ...acc,
-                            [emergency.emergency]: Number(item.contextIndicatorValue),
+                            [emergency.emergency]: item.format === 'percent'
+                                ? decimalToPercentage(Number(item.contextIndicatorValue))
+                                : Number(item.contextIndicatorValue),
```
Comment (subinasr) [inline app/views/Dashboard/Country/index.tsx:759]: Let's add a FIXME for after this type is fixed from server side.

File: app/views/Dashboard/Country/index.tsx [hunk 2]
```diff
@@ -824,7 +828,7 @@ function Country(props: Props) {
     const statusRendererParams = useCallback((_, data: CountryWiseOutbreakCases) => ({
         heading: data.emergency,
         // TODO: fetch format from server
```
Comment (subinasr) [inline app/views/Dashboard/Country/index.tsx]: Remove this TODO if this is already implemented.

File: app/views/Dashboard/Country/index.tsx [hunk 3]
```diff
@@ -612,6 +613,7 @@ function Country(props: Props) {
                 id: total.id,
                 emergency: total.emergency,
                 totalCases: total.indicatorValue,
+                format: total.format as FormatType,
```
Comment (subinasr) [inline app/views/Dashboard/Country/index.tsx:616]: I don't think this cast is necessary. Please check.

File: app/views/Dashboard/Overview/MapView/MapModal/index.tsx [hunk 4]
```diff
@@ -162,13 +162,12 @@ interface ModalProps {
...
     indicatorMonth?: string;
     format?: FormatType;
-
     indicatorValue?: number;
     indicatorId: string;
     outbreakId: string | undefined;
-
     indicatorExplicitlySet: boolean;
     selectedIndicatorType: IndicatorType | undefined;
+    filterValues?: FilterType
```
Comment (subinasr) [inline app/views/Dashboard/Overview/MapView/MapModal/index.tsx]:
```suggestion
    filterValues?: FilterType;
```

Comment (subinasr) [reply app/views/Dashboard/Overview/MapView/MapModal/index.tsx]: Also, do we need this?

**Review by subinasr — CHANGES_REQUESTED** [review]

File: app/views/Dashboard/Country/index.tsx (hunk 3 above)
Comment (shreeyash07) [reply app/views/Dashboard/Country/index.tsx:616]: Its needed

**Review by subinasr — APPROVED** [review]

**Pull request merged**